### DISCUSSION TITLE
Supporting POST for REST Search Queries would be nice for Roblox consumers

### DISCUSSION THREAD

**Discussion by jakedowns** [directus/directus#16256] [Feature Requests]
> There's a lot of discussion around whether or not to put a body in a GET request, to use POSTs to create search queries, or to rely on a different method altogether. As of right now, we've chosen [to align with IETF's HTTP SEARCH Method specification](https://datatracker.ietf.org/doc/draft-ietf-httpbis-safe-method-w-body). While we recognize this is still a draft spec, the SEARCH method has been used extensively before in the WebDAV world ([spec](https://tools.ietf.org/html/rfc5323)), and compared to the other available options, it feels like the "cleanest" and most correct to handle this moving forward. As with everything else, if you have any ideas, opinions, or concerns, [we'd love to hear your thoughts](https://github.com/directus/directus/discussions/new).

the roblox HttpService doesn't appear to have a way to manually set the method to SEARCH, exposing only GET and POST via it's api.

https://create.roblox.com/docs/reference/engine/classes/HttpService

**Comment by rijkvanzanten** [comment]
You can still use GET for reads though!

**Comment by jakedowns** [comment]
yeah luckily our query strings aren't too long on this project.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
